charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -164,6 +164,9 @@ data:
         client:
           resources: {{- .Values.global.dagOnlyDeployment.resources | toYaml | nindent 12 }}
         {{- end }}
+        {{- if .Values.global.dagOnlyDeployment.persistence }}
+        persistence: {{- .Values.global.dagOnlyDeployment.persistence | toYaml | nindent 10 }}
```

Comment:
Do we want to add a default value for this? Even something like `[]` or `{}` ?

Reply:
persistence: {} should be fine

Reply:
with this change https://github.com/astronomer/airflow-chart/blob/master/values.yaml#L555-L560 these sections are now overridable